GenderPayGap.WebUI/Search/SearchRepository.cs

```diff
@@ -72,7 +72,14 @@ private static List<SearchCachedOrganisation> LoadAllOrganisations(IDataReposito
                         EmployerReference = o.EmployerReference?.Trim(),
                         OrganisationNames = o.OrganisationNames.OrderByDescending(n => n.Created).Select(on => new SearchReadyValue(on.Name)).ToList(),
                         MinEmployees = o.GetLatestReturn()?.MinEmployees ?? 0,
-                        Status = o.Status
+                        Status = o.Status,
+                        OrganisationSizes = o.Returns.Select(r => r.OrganisationSize).ToList(),

```

Comment:
* It would be good to only take the Submitted Returns, rather than old versions
* This will often be the same for all 3-4 years - it would be nice to remove duplicates

so maybe `o.Returns.Where(r => r.Status == ....Active).Select(r => r.OrganisationSize).Distinct().ToList()`